pandas/tests/computation/test_eval.py

```diff
@@ -1757,6 +1757,29 @@ def test_no_new_globals(self, engine, parser):
         assert gbls == gbls2
 
 
+class TestEvalNamedKWargs(object):
```

Comment:
I don't think we need a class here; OK to remove this and put tests at top level of module